app/pathogen/arbovirus/wip/page.tsx

```diff
@@ -0,0 +1,39 @@
+"use client"
+import { useScrollSectionGroup } from "@/components/ui/scroll-section-group/use-scroll-section-group";
+import { notFound, useSearchParams } from "next/navigation";
+import React, { useEffect } from "react";
+import { redesignedArbovirusPageSections, sectionUrlParameterToSectionId } from "./sections";
+
+export default function RedesignedArbovirusPage() {
+  const searchParams = useSearchParams();
+
+  const {renderScrollSectionGroup, moveScrollSectionGroupToSection} = useScrollSectionGroup({
+    scrollSectionGroupProps: {
+      sections: redesignedArbovirusPageSections,
+      className:"col-span-10 row-span-2",
+      scrollThrottleThresholdMilliseconds: 700
+    }
+  })
+
+  useEffect(() => {
+    const sectionUrlParameter = searchParams.get('section');
+    const sectionId = sectionUrlParameterToSectionId({urlParam: sectionUrlParameter})
+
+    if(sectionId) {
+      moveScrollSectionGroupToSection({ sectionId });
+    }
+  }, [searchParams])
+
+  if(process.env.NEXT_PUBLIC_WEBSITE_REDESIGN_ENABLED !== 'true') {
+    return notFound();
+  }
```

Comment:
The `NEXT_PUBLIC_...` in front of the variable name is non-negotiable apparently if we want to use it in client side files (which kind of makes sense). This feature flag is just so that people in production don't see my colourful page.

@HimanshuRanka if you could please set this environment variable to `true` in the preview environment but to anything that isn't `true` in the production environment just so that people can see my page on their vercel previews I would really appreciate it :heart: